src/AutoComplete/AutoComplete.js

```diff
@@ -151,6 +151,13 @@ class AutoComplete extends Component {
      */
     onUpdateInput: PropTypes.func,
     /**
+     * This function is called to retrieve the text from the `chosenRequest` and should return a string.
+     * By default, if the `chosenRequest` is a string, that will be used.
+     * Otherwise the key specified in `dataSourceConfig.text` will be returned
+     * for the `chosenRequest`.
+     */
+    chosenRequestText: PropTypes.func,
```

Comment:
Add the prop types declaration alphabetically. So, move the declaration of `chosenRequestText` prop type after `animated` prop type, line 56